openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -88,32 +88,128 @@
 - control_key: AU-3
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        When the OpenShift API audit system is enabled and configured it generates
+        audit records containing information that establishes the type of event that occurred,
+        when the event occurred, where the event occurred, the source of the event,
+        the outcome of the event, and the idenity of the individuales or subjects associated
+        with the event.  Note,  usage of - oc debug - does not create audit events.
+
+        OpenShift Container Platform cluster logging is designed to be used with the default
```

Comment:
I don't think we need lines 100-121. AU-3 is about the audit record itself containing the required traceability information mentioned in line 94.